### DISCUSSION TITLE
Failed to download extension "odbc"

### DISCUSSION THREAD

**Discussion by AINIVERSHERRY** [duckdb/duckdb#13403] [Q&A]
### What happens?

I want to query data in SQL Server through duckDB.

When I INSTALL odbc, got error: Failed to download extension "odbc" at URL "http://extensions.duckdb.org/v0.10.3/windows_amd64/odbc.duckdb_extension.gz".

Where can I find odbc.duckdb_extension.gz file?

### To Reproduce

conn = duckdb.connect()
conn.execute("INSTALL odbc;")
conn.execute("LOAD odbc;")

### OS:

x64

### DuckDB Version:

v0.10.3

### DuckDB Client:

Python

### Full Name:

金毅轩

### Affiliation:

Pharbers

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have tested with a stable release

### Did you include all relevant data sets for reproducing the issue?

Yes

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Python version, Linux distribution) to reproduce the issue?

- [X] Yes, I have

**Comment by szarnyasg** [comment]
Hi @AINIVERSHERRY, ODBC support is provided as a driver (not as an extension of DuckDB). See https://duckdb.org/docs/api/odbc/overview and https://github.com/duckdb/duckdb/releases/download/v1.0.0/duckdb_odbc-windows-amd64.zip.
